CMakeLists.txt

```diff
@@ -56,6 +56,12 @@ include_directories (
 	${PROJECT_BINARY_DIR}/Configuration
 	${SERVER_INCLUDE_DIRECTORIES}
 	)
+	
```

Comment:
Is this change necessary?

Reply:
Yep - necessary. Needed to move the resolution of the OPCUA_TOOLKIT_LIBS variable (based on whatever is defined in the build config) before the inclusion of the Meta sub-directory. Without this Meta doesn't know what the toolkit libs are to publish via BuildInformation.

The block above the new location of OPCUA_TOOLKIT_LIBS  resolution (not moved in this branch) already references variable OPCUA_TOOLKIT_PATH - defined in the build config.